Makefile

```diff
@@ -30,6 +30,8 @@ deps: clean
 	yarn
 	# Call the build script for the adapter if defined (TypeScript adapters have this extra build/compile step)
 	# We use `wsrun` to build workspace dependencies in topological order (TODO: use native `yarn workspaces foreach -pt run build` with Yarn 2)
+	if [ $(adapter) = bootstrap ]; then yarn wsrun -mre -p @chainlink/ea-bootstrap -t build; fi
+	if [ $(adapter) = external-adapter ]; then yarn wsrun -mre -p @chainlink/external-adapter -t build; fi
```

Comment:
Yeah this could be fixed by adding `-adapter` to the package names and changing the github strategies to say `ea-bootstrap` instead of `bootstrap`

#174